idrstream_cp_env.yml

```diff
@@ -0,0 +1,30 @@
+name: idrstream_cp
+
+channels:
+- conda-forge
+
+dependencies:
+- conda-forge::python=3.8.13
+- conda-forge::jupyter=1.0.0
+- conda-forge::pandas=1.4.2
+- conda-forge::pytorch=1.11.0
+- conda-forge::scikit-image=0.19.3
+- conda-forge::pyimagej=1.2.1
+- conda-forge::pip=22.1.2
+- conda-forge::numpy=1.23.3
+- conda-forge::matplotlib
```

Comment:
I think it would be worth adding version numbers to these next 6 libraries to help with reproducibility. 

If you can check which versions you have in your `idrstream_cp` conda environment you can add those versions here, make a temp conda env from the new yml file, and test the test env on the example notebook.